Makefile

```diff
@@ -69,10 +69,7 @@ compile-typespec:
 		$(SERVICE).spec
 	rm lib/biokbase/workspace/authclient.py
 
-test: test-client test-service
-
-test-client: test-service
-	$(ANT) test_client_import
```

Comment:
Testing that the client built correctly made sense back when the Workspace was the first and only Java SDK service. Now it's a waste of time